src/DataGrid.tsx

```diff
@@ -1146,7 +1172,8 @@ function DataGrid<R, SR, K extends Key>(
         rootClassname,
         {
           [viewportDraggingClassname]: isDragging,
-          [cellAutoResizeClassname]: autoResizeColumn !== null
+          [cellAutoResizeClassname]:
```

Comment:
Maybe this a more appropriate name
```suggestion
          [autosizeColumnsClassname]:
```